embulk-output-jdbc/build.gradle

```diff
@@ -6,3 +6,42 @@ embulkPlugin {
     category = "output"
     type = "jdbc"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "takumakanari"
+                        email = "chemtrails.t@gmail.com"
+                    }
+                    developer {
+                        name = "Tomohiro Hashidate"
+                        email = "kakyoin.hierophant@gmail.com"
+                    }
```

Comment:
@joker1007 Hi! We're going to nominate you here, too, as the same as https://github.com/embulk/embulk-input-jdbc/pull/210#discussion_r639415928.

Reply:
Thanks!!
It's no problem.